core/src/main/scala/kafka/admin/AdminUtils.scala

```diff
@@ -602,13 +603,14 @@ object AdminUtils extends Logging with AdminUtilities {
                 configTup match {
                   case (k: String, v: String) =>
                     props.setProperty(k, v)
-                  case _ => throw new IllegalArgumentException(s"Invalid ${entityConfigPath} config: ${str}")
+                  case _ => throw new IllegalArgumentException(s"Invalid ${entityConfigPath} config: ${data}")
                 }
-            case _ => throw new IllegalArgumentException(s"Invalid ${entityConfigPath} config: ${str}")
+            case _ => throw new IllegalArgumentException(s"Invalid ${entityConfigPath} config: ${data}")
           }
 
-        case _ => throw new IllegalArgumentException(s"Unexpected value in config:(${str}), entity_config_path: ${entityConfigPath}")
+        case _ => throw new IllegalArgumentException(s"Unexpected value in config:(${data}), entity_config_path: ${entityConfigPath}")
       }
+      case None =>
```

Comment:
Btw, you can use `foreach` instead of pattern matching so that you don't need this empty branch.